cmd/dashboard/main.go

```diff
@@ -47,6 +47,7 @@ var (
 	readOnly           = flag.Bool("read-only", false, "Enable or disable read only mode")
 	webDir             = flag.String("web-dir", "", "Dashboard web resources dir")
 	csrfSecureCookie   = flag.Bool("csrf-secure-cookie", true, "Enable or disable Secure attribute on the CSRF cookie")
+	tenantNamespace    = flag.String("namespace", "", "Tenant namespace. If set, limits the scope of resources watched to this namespace")
```

Comment:
It's not just `watched` right, it's anything? E.g. I can create/rerun resources etc in the tenant ns only?

Reply:
Good catch, will update the description.

Reply:
fixed

Reply:
@a-roberts in fact no, it's really for watching (and driving the frontend).

> I can create/rerun resources etc in the tenant ns only?

This is not taken care of by this flag but by RBAC rules.

It should be noted somewhere (at least in the docs). Not sure about the correct description to set here.